setup.py

```diff
@@ -23,9 +23,10 @@
 from setuptools import setup, find_packages
 import json
 
-bowerJson = json.loads(open("bower.json").read())
-if bowerJson:
-    version = bowerJson.get('version')
+
+packageJson = json.loads(read("package.json"))
```

Comment:
We use snake_case in Python :)

```suggestion
package_json = json.loads(read("package.json"))
```